stumpy/mstump.py

```diff
@@ -130,6 +130,9 @@ def _multi_mass(Q, T, m, M_T, Σ_T, μ_Q, σ_Q):
     σ_Q : numpy.ndarray
         Standard deviation of `Q`
 
+    T_subseq_isconstant : numpy.ndarray
+        A boolean array that indicates whether a subsequence in `T_A` is constant (True)
```

Comment:
subsequence in `T` is constant. Not `in T_A`

Reply:
Note that in the current implementation of the `mstump` algorithms, I had purposely coded everything in anticipation that, one day, we'd somehow allow multi-dimensional AB-joins (not sure what the results would mean). If possible, internally, I feel like we should continue to assume that there are two sets of of multi-dimensional time sereis `T_A` and `T_B` and only the API is currently only supporting `T`. So, we may want to keep things as, say, `T_A_subseq_isconstant` and `T_B_subseq_isconstant` where appropriate

Reply:
> subsequence in `T` is constant. Not `in T_A`

I tried to make it similar to the existing description of `M_T` and `Σ_T` in this docstring. Should I revise those as well?

Reply:
> Note that in the current implementation of the `mstump` algorithms, I had purposely coded everything in anticipation that, one day, we'd somehow allow multi-dimensional AB-joins (not sure what the results would mean). If possible, internally, I feel like we should continue to assume that there are two sets of of multi-dimensional time sereis `T_A` and `T_B` and only the API is currently only supporting `T`. So, we may want to keep things as, say, `T_A_subseq_isconstant` and `T_B_subseq_isconstant` where appropriate

Ahhhhh..got it!!! I will generalize it by passing both `T_A_subseq_isconstant` and `T_B_subseq_isconstant` to the function. Cool :)

Reply:
@seanlaw 


> > subsequence in `T` is constant. Not `in T_A`
> 
> I tried to make it similar to the existing description of `M_T` and `Σ_T` in this docstring.
> 
> **Should I revise those as well?**


Would you please let me know about this? I am 99% sure that I need to revise the description of `M_T` and `Σ_T`. But, I just wanted to check it with you as well.

Reply:
> I will generalize it by passing both T_A_subseq_isconstant and T_B_subseq_isconstant to the function. 

In this function, it looks like it is only a single multi-dimensional `Q` and and multi-dimensional `T`. And so `T_subseq_isconstant` is fine

> I am 99% sure that I need to revise the description of M_T and Σ_T.

Yes, please change to `T`.